usr/share/lib/img_proof/tests/SLES/Azure/test_sles_azure_kernel_cmdline.py

```diff
@@ -9,7 +9,6 @@ def test_sles_azure_kernel_cmdline(host, determine_architecture):
             'earlyprintk=ttyS0'
         ],
         'AARCH64': [
-            'console=tty1',
```

Comment:
This is not right, the image description contains `console=tty1` as part of the kernel cmdline setup. If this disappears we need to find out why.